src/vmm/src/vmm_config/hotplug.rs

```diff
@@ -1,19 +1,20 @@
 // Copyright 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-use serde::{Deserialize, Serialize};
 use std::io;
+
+use serde::{Deserialize, Serialize};
+
 use crate::vstate::vcpu::VcpuError;
 use crate::StartVcpusError;
 
 /// Firecracker aims to support small scale workloads only, so limit the maximum
 /// vCPUs supported.
-pub const MAX_SUPPORTED_VCPUS: u8 = 32;
```

Comment:
nit: this should already not have been introduced in the first commit :) Try to avoid adding code in one commit and then deleting it again in a follow up (unless necessary of course!)